orchestra/tests/test_dashboard.py

```diff
@@ -470,11 +472,12 @@ def test_assignment_snapshot_upgrades(self):
         good_snapshots = {
             'snapshots': [{
                 'data': {},
-                'datetime': datetime.utcnow().isoformat(),
+                'datetime': (
+                    datetime.utcnow().replace(tzinfo=tzutc()).isoformat()),
```

Comment:
this is JSON and not django---do you still think we should use django tooling for it?